scripts/blocklist-skylink.sh

```diff
@@ -3,14 +3,49 @@
 set -e # exit on first error
 
 if [ -z "$1" ]; then
-    echo "Please provide a skylink to blocklist" && exit 1
+    echo "Please provide either a skylink or file with skylinks separated by blank lines" && exit 1
 fi
 
+#########################################################
+# read either a file containing skylinks separated by new 
+# lines or a single skylink and put them in an array
+#########################################################
+skylinks=()
+if test -f "$1"; then
+    OLDIFS=$IFS
+    IFS=','
+    while read line
+    do
+        if [[ $line =~ ([a-zA-Z0-9_-]{46}) ]]; then
+            skylinks+=("$BASH_REMATCH")
+        else
+            echo "Incorrect skylink: $line" && exit 1
```

Comment:
Would it make sense to also print the line on which the bad skylink sits? That would help a little with vim. Although we can also just search for it... I don't know, what do you think?

Reply:
added